packages/IMAPClient-UI.package/ICFolderDialog.class/instance/openEmail..st

```diff
@@ -1,8 +1,8 @@
 action - emails
 openEmail: anICEmail
 	
-	[anICEmail open.	
+	"["anICEmail open.
 	self selectedFolder retrieveFlagsFor: anICEmail.
-	self triggerEmailChanges]
-	on: Error
-	do: [:e | self inform: self class loadEmailBodyFailure]
\ No newline at end of file
+	self triggerEmailChanges"]
+	on: Error ConnectionClosed 
+	do: [:e | self inform: self class loadEmailBodyFailure]"
```

Comment:
commented out lines should be removed if they are not needed anymore

Reply:
This might just be a quick fix for the errors that appear because we haven't implemented flags in the offline mode yet. So in this case we might still need the commented out lines.